scripts/cluster_network.py

```diff
@@ -140,6 +140,9 @@
 from pypsa.networkclustering import (busmap_by_kmeans, busmap_by_spectral_clustering,
                                      _make_consense, get_clustering_from_busmap)
 
+import warnings
+warnings.filterwarnings(action='ignore', category=UserWarning)
```

Comment:
Out of curiosity - what warning is suppressed?

Reply:
KMeans user warning that dataset contains features that are not used in the clustering.

Reply:
Maybe @martacki can comment on this deprecation warning and whether it's something we need to address?